plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/procedure/VacuumProcedure.java

```diff
@@ -137,6 +138,9 @@ public void vacuum(
         catch (TrinoException e) {
             throw e;
         }
+        catch (IOException e) {
+            throw new TrinoException(DELTA_LAKE_FILESYSTEM_ERROR, format("Failure when vacuuming %s.%s with retention %s: %s", schema, table, retention, e), e);
+        }
         catch (Exception e) {
```

Comment:
This should be changed to `catch (RuntimeException)` as there are no other checked exceptions to catch.